packages/chrome/src/elements/subnav/SubNavItem.tsx

```diff
@@ -11,6 +11,7 @@ import { StyledSubNavItem } from '../../styled';
 import { useChromeContext } from '../../utils/useChromeContext';
 
 interface ISubNavItemProps {
+  /** Determines if the current item is in the sub-navigation */
```

Comment:
It looks like we chose to use the term "subnav" here: https://garden.zendesk.com/components/chrome#navigation-panel. Let's remain consistent and replace "sub-navigation" with "subnav".

Reply:
Fixed in latest update.